logstash/pipeline/logstash.conf

```diff
@@ -1,7 +1,8 @@
 input {
-	tcp {
-		port => 5000
-	}
+    tcp {
+        port  => 5000
+        codec => json
```

Comment:
I think that may be logspout specific. I'm having problems testing the current setup, but I'll ensure things run and see if we can do without that line.